crates/ln-dlc-node/src/tests/system_tests/channel_less_payment.rs

```diff
@@ -0,0 +1,154 @@
+use bitcoin::Address;
+use std::str::FromStr;
+use std::time::Duration;
+// Related issue: https://github.com/get10101/10101/issues/55.
+use crate::tests::system_tests::create_tmp_dir;
+use crate::tests::system_tests::fund_and_mine;
+use crate::tests::system_tests::has_channel;
+use crate::tests::system_tests::init_tracing;
+use crate::tests::system_tests::log_channel_id;
+use crate::tests::system_tests::setup_ln_node;
+
+#[tokio::test]
+async fn given_no_channel_with_coordinator_when_invoice_generated_then_can_be_paid_through_coordinator(
+) {
+    init_tracing();
+
+    let test_dir = create_tmp_dir("multi_hop_test");
+
+    // 1. Set up three LN-DLC nodes.
+    let alice = setup_ln_node(&test_dir, "alice").await;
+    let coordinator = setup_ln_node(&test_dir, "coordinator").await;
+    let bob = setup_ln_node(&test_dir, "bob").await;
+
+    tracing::info!("alice: {}", alice.info);
+    tracing::info!("coordinator: {}", coordinator.info);
+    tracing::info!("bob: {}", bob.info);
+
+    let _alice_bg = alice.start().await.unwrap();
+    let _coordinator_bg = coordinator.start().await.unwrap();
+    let _bob_bg = bob.start().await.unwrap();
+
+    // 2. Connect the two nodes.
+
+    tokio::time::sleep(Duration::from_secs(2)).await;
+    alice.keep_connected(coordinator.info).await.unwrap();
+    bob.keep_connected(coordinator.info).await.unwrap();
+    alice.keep_connected(bob.info).await.unwrap();
```

Comment:
🤔 I guess `alice` and `bob` don't have to be connected for this test to work. do they?